kmath-core/src/commonMain/kotlin/scientifik/kmath/operations/NumberAlgebra.kt

```diff
@@ -40,12 +48,13 @@ interface ExtendedField<T> : ExtendedFieldOperations<T>, Field<T> {
  * TODO inline does not work due to compiler bug. Waiting for fix for KT-27586
  */
 inline class Real(val value: Double) : FieldElement<Double, Real, RealField> {
+    override val context: RealField
```

Comment:
Does not make a lot of difference. I think the variant from before was more readable.